libs/eo/transfers/src/lib/eo-transfers-table.component.ts

```diff
@@ -164,32 +177,45 @@ interface EoTransferTableElement extends EoListedTransfer {
     </watt-paginator>
     <ng-template #notActive><watt-badge type="neutral">Inactive</watt-badge></ng-template>
 
-    <eo-transfers-modal title="New transfer agreement"></eo-transfers-modal>
-    <eo-transfers-drawer></eo-transfers-drawer>
+    <eo-transfers-create-modal title="New transfer agreement"></eo-transfers-create-modal>
+    <eo-transfers-drawer
+      [transfer]="selectedTransfer"
+      (closed)="transferSelected.emit(undefined)"
```

Comment:
Question: Do we need send an undefined with this emit? :) I haven't used that before myself, so didn't know it was something you did